charters/DataStore/charter.md

```diff
@@ -0,0 +1,75 @@
+
+### Core capabilities
+* DSS data model and lifecycle (versioned bundles, etc) - Ongoing support and maintenance of the implementation of the data model. 
+* Subscriptions/Eventing - Implementation of Data lifecycle web-hooks (new bundle, new file, delete bundle, delete file). The Data Store implementation will move away from the current dependance on Elastic Search Percolate for our event subsystem. Eventing will depend instead on the AWS and GCP cloud infrastructure directly.
+* Multi-cloud replication of objects - There are three parts to this:
+   1. Maintenance and improvements to the synchronization implementation between AWS and GCP
+   2. Extending the cloud support to more vendors (such as Microsoft Azure) 
+   3. Supporting multiple replicas within a single cloud.
+* Support for plug-able indexes - Provide a standard interface for connecting indexing modules to the Data Store. This interface will provide a mechanism to connect indexing subsystems to receive events about the data. 
+
+### Security
+* User authentication system implementation
+* Data access authorization system implementation 
+* DevSecOps - implementation of features required for eventual FISMA moderate deployments (authentication, authorization, logging, auditing, etc).
+* Operations for DSS - Implement and configure tools to facilitate the operation of the Data Store service in a production environment
+
+### Community engagement
+* Triage and integration of feature requests from the community into the Data Store roadmap. 
+* Outreach and engagement of the community
+* Training
+* Hackathons
+
+
+## Out-of-scope
+* Other index/query methods/engines - we should implement these as stand-alone projects against modular index/query API.
+* Matrix service API
+
+## Milestones
+* Mid-2018:  1000 bundle test scale, deploy as part of HCA DCP Pilot
+* EOY 2018: add checkout, collections, improved scaling/hardening, generic events to support stand-alone indexers, additional gaps identified in HCA DCP Pilot.
+* Future: (not in order of precedence) native GCP support, Authorization support for controlled-access data, additional scale/hardening, Biosphere requirements, tiered storage, content zones, FISMA moderate capabilities, single-replica deployments
+
+## Roles
+
+### Project Lead 
+[Brian O’Connor](mailto:brocono@ucsc.edu) 
+
+### Product Owner 
+[Kevin Osborn](mailto:kosborn2@ucsc.edu) 
+
+### Technical Lead 
+[Hannes Schmidt](mailto:hannes@ucsc.edu) 
+
+## Communication
+### Slack Channels
+* HumanCellAtlas/data-store : general data store discussions
+* HumanCellAtlas/data-store-eng : development discussions
+### Mailing list(s)
```

Comment:
If not used, then **Mailing list(s)** and **Discussion Forum(s)** can be deleted.